js/application.js

```diff
@@ -1,138 +1,245 @@
+      var ran = Math.floor(Math.random()*pack.length);
+      var cardPicked = pack[ran];
+
+      //remove the card picked from the pack
+      pack.splice(ran, 1);
+      $('#cards_left').text('Cards left :' + pack.length)
+      
+      //display the card picked in the game_playing_section
+      $('.card_picked_type').text(cardPicked.nameCard);
+      $('.card_picked_rule').text(cardPicked.ruleName);
+      $('.card_picked_description').text(cardPicked.ruleDescription);
+
+      //Make a rule case
+      if(cardPicked.ruleName === "Make a Rule"){
+        $('.pick_card_phase').hide();
+        $('.make_a_rule_case').show();
+      }else{
+      // change the pick button to the next player button
+      $('.pick_card_phase').fadeOut(0);
+      $('.next_player_phase').fadeIn(2000);
+      }
+    }
+    
+    // WHEN THERE IS NO MORE CARDS THE GAME ENDS
+    else{
+
+      //EFFECTS
+      $('#pick_button').hide();
+      $('.add_sips').fadeOut(1500);
+      $('#game_playing_section').slideUp(2000);
+      $('#final_score_section').slideDown(2000);
+
+      //CREATES THE FINAL SCORE TABLE
+      for (var i = 0; i < players.length; i++){
+        var finalName = players[i].namePlayer;
+        var finalScore = players[i].score;
+        var finalRow = '<tr><td>' + finalName + '</td><td class="score_displayed">' + finalScore + '</td></tr>';
+        $(finalRow).appendTo(".final_score_table").hide().fadeIn(6000);
+      };
+    }
+  cardFlipped = true;
+  }
+})
 
-		//pick a card from the pack
-  	var ran = Math.floor(Math.random()*pack.length);
-		var cardPicked = pack[ran];
+//MAKE A RULE CASE GO BACK
+$('#finish_rule').click(function(){
+  $('.make_a_rule_case').hide();
+  $('.next_player_phase').fadeIn(2000);
 
-		//remove the card picked from the pack
-		pack.splice(ran, 1);
-		
-		//display the card picked in the game_playing_section
-		$('#card_picked_rule').text(cardPicked.ruleName);
-		$('#card_picked_description').text(cardPicked.ruleDescription);
+  var contentRuleCreated = $('#input_make_a_rule').val();
+  var newRule = '<p>' + contentRuleCreated + '</p>';
+  $('#rules_added').hide().fadeIn(1000);
+  $(newRule).appendTo('#rules_added').hide().fadeIn(2000);
 })
 
-//START THE GAME
-$('#start_game').click(function(){
-
-	//CREATE SCORE TABLE
-	for (var i = 0; i < players.length; i++){
-		var getName = players[i].namePlayer; 
-		var getScore = players[i].score;
-		var newRow = '<tr><td>' + getName + '</td><td class="score_displayed">' + getScore + '</td><td><button class="add_sips">+</button></td></tr>';
-		$(newRow).appendTo(".score_table").hide().fadeIn(5000);
-	}
-	$('#game_setting_section').fadeOut(2000);
+//NEXT PLAYER TO PLAY
+var listTurnQuotes = [', your turn!',' dig in!',', you\'re up!',' WAKE UP!', ', the others are thirsty.',', breathe... you can do this.'];
+var yourTurnQuote = function(){
+  var turnQuote = listTurnQuotes[Math.floor(Math.random()*listTurnQuotes.length)];
+  return turnQuote;
+};
+
+var a;
+var j = 1;
```

Comment:
same logic for `a` and `j`, should be inside the function to limit it's visibility (other functions won't be able to see/change that variable)